Source/WebCore/page/scrolling/ScrollingCoordinator.cpp

```diff
@@ -368,7 +368,9 @@ bool ScrollingCoordinator::shouldUpdateScrollLayerPositionSynchronously(const Lo
 ScrollingNodeID ScrollingCoordinator::uniqueScrollingNodeID()
 {
     static ScrollingNodeID uniqueScrollingNodeID = 1;
-    return uniqueScrollingNodeID++;
+    auto returnID = uniqueScrollingNodeID;
+    uniqueScrollingNodeID = ProcessQualified(returnID.object() + 1);
```

Comment:
This is a bit of a hack. I think we should go the whole way and make ScrollingNodeID be a `ProcessQualified<ObjectIdentifer<>>`

Reply:
Fixed.